code/modules/chemistry/Reagents-PoisonEtc.dm

```diff
@@ -1135,6 +1135,50 @@ datum
 				..()
 				return
 
+		harmful/hydrogen_sulfide
+			name = "hydrogen sulfide"
+			id = "hydrogen_sulfide"
+			description = "A dangerous asphyxiant substance that inhibits celular respiration."
```

Comment:
"cellular" in english spelling; otherwise this looks neato

Reply:
curse you english language!!!